### PR TITLE
add data_release_7.0

### PR DESCRIPTION
**Pull request by lindaxiang**

### CODE REVIEW

**Review by justincorrigible — APPROVED** [review]

File: docs/release-notes/data-releases.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ platform_key: DOCS_DATA_RELEASES
 
 An ARGO data release is a curated data set of clinical and molecular data submitted to the ARGO Data Platform. Data releases happen approximately four times a year and are cumulative in nature. Released data can be browsed using the [File Repository](https://platform.icgc-argo.org/repository) and [downloaded](/docs/data-access/data-download) using a client tool, provided that access to controlled data has been granted. To access controlled data, please see the [DACO application process here](/docs/data-access/daco/applying).
 
+## Data Release 7.0
+
+**Release Date:** November 06, 2023
+
+#### New Updates
+
+ICGC ARGO is excited to announce the first release of **new clinical data** available now for Australian Pancreatic Cancer Initiative [(APGI-AU)](https://www.icgc-argo.org/page/145/apgi-au), pancreatic cancer [(PACA-CA)](https://www.icgc-argo.org/page/96/panchope), and papillary thyroid cancer [(PTC-SA)](https://www.icgc-argo.org/page/98/ptcp). APGI-AU was a member of the original ICGC-25K data portal as PACA-AU.
```
Comment (justincorrigible) [inline docs/release-notes/data-releases.md:15]: should it be APCI? (it says APGI)

Comment (lindaxiang) [reply docs/release-notes/data-releases.md:15]: Good catch @justincorrigible . sorry it should be `Australian Pancreatic Genome Initiative`. I will correct it.

Comment (justincorrigible) [reply docs/release-notes/data-releases.md]: At risk of being annoyingly pedantic, I believe it's actually `Australian Pancreatic Cancer Genome Initiative`. no?
https://www.pancreaticcancer.net.au/

But maybe this is a different one?

**Comment by lindaxiang** [comment]
@justincorrigible you are correct, APGI-AU stands for Australian Pancreatic Cancer Genome Initiative. The link I put there is the one on ARGO org site.

File: docs/release-notes/data-releases.md [hunk 2]
```diff
@@ -6,13 +6,27 @@ platform_key: DOCS_DATA_RELEASES
 
 An ARGO data release is a curated data set of clinical and molecular data submitted to the ARGO Data Platform. Data releases happen approximately four times a year and are cumulative in nature. Released data can be browsed using the [File Repository](https://platform.icgc-argo.org/repository) and [downloaded](/docs/data-access/data-download) using a client tool, provided that access to controlled data has been granted. To access controlled data, please see the [DACO application process here](/docs/data-access/daco/applying).
 
+## Data Release 7.0
+
+**Release Date:** November 06, 2023
+
+#### New Updates
+
+ICGC ARGO is excited to announce the first release of **new clinical data** available now for Australian Pancreatic Genome Initiative [(APGI-AU)](https://www.icgc-argo.org/page/145/apgi-au), Pancreatic Cancer [(PACA-CA)](https://www.icgc-argo.org/page/96/panchope), and Papillary Thyroid Cancer [(PTC-SA)](https://www.icgc-argo.org/page/98/ptcp). APGI-AU was a member of the original ICGC-25K data portal as PACA-AU.
```
Comment (justincorrigible) [inline docs/release-notes/data-releases.md:15]:
```suggestion
ICGC ARGO is excited to announce the first release of **new clinical data** available now for Australian Pancreatic Cancer Genome Initiative [(APGI-AU)](https://www.icgc-argo.org/page/145/apgi-au), Pancreatic Cancer [(PACA-CA)](https://www.icgc-argo.org/page/96/panchope), and Papillary Thyroid Cancer [(PTC-SA)](https://www.icgc-argo.org/page/98/ptcp). APGI-AU was a member of the original ICGC-25K data portal as PACA-AU.
```

File: docs/release-notes/data-releases.md [hunk 3]
```diff
@@ -6,13 +6,27 @@ platform_key: DOCS_DATA_RELEASES
...
+
 ## Data Release 6.0
 
 **Release Date:** April 05, 2023
 
 #### New Updates
 
-ICGC ARGO is excited to announce the first release of a **new** RNA-Seq analysis workflow: [RNA Seq Alignment](/docs/analysis-workflows/rna-alignment), available now for transcriptome sequencing data from Australian Pancreatic Cancer Initiative (APGI-AU). APGI-AU was a member of the original ICGC-25K data portal as PACA-AU. While clinical data is being actively submitted by the program, a selection of its RNA-Seq data has been reprocessed against the latest GRCh38 Human Reference Genome using the ARGO RNA Seq Alignment Workflow.
+ICGC ARGO is excited to announce the first release of a **new** RNA-Seq analysis workflow: [RNA Seq Alignment](/docs/analysis-workflows/rna-alignment), available now for transcriptome sequencing data from Australian Pancreatic Genome Initiative (APGI-AU). APGI-AU was a member of the original ICGC-25K data portal as PACA-AU. While clinical data is being actively submitted by the program, a selection of its RNA-Seq data has been reprocessed against the latest GRCh38 Human Reference Genome using the ARGO RNA Seq Alignment Workflow.
```
Comment (justincorrigible) [inline docs/release-notes/data-releases.md:29]:
```suggestion
ICGC ARGO is excited to announce the first release of a **new** RNA-Seq analysis workflow: [RNA Seq Alignment](/docs/analysis-workflows/rna-alignment), available now for transcriptome sequencing data from Australian Pancreatic Cancer Genome Initiative (APGI-AU). APGI-AU was a member of the original ICGC-25K data portal as PACA-AU. While clinical data is being actively submitted by the program, a selection of its RNA-Seq data has been reprocessed against the latest GRCh38 Human Reference Genome using the ARGO RNA Seq Alignment Workflow.
```

**Comment by justincorrigible** [comment]
Preview available here https://65456bb0d1bd760008620284--argo-docs.netlify.app/docs/release-notes/data-releases#data-release-70

File: docs/release-notes/data-releases.md [hunk 4]
```diff
@@ -6,13 +6,27 @@ platform_key: DOCS_DATA_RELEASES
...
+
+**Release Date:** November 06, 2023
+
+#### New Updates
+
+ICGC ARGO is excited to announce the first release of **new clinical data** available now for Australian Pancreatic Cancer Genome Initiative [(APGI-AU)](https://www.icgc-argo.org/page/145/apgi-au), Pancreatic Cancer [(PACA-CA)](https://www.icgc-argo.org/page/96/panchope), and Papillary Thyroid Cancer [(PTC-SA)](https://www.icgc-argo.org/page/98/ptcp). APGI-AU was a member of the original ICGC-25K data portal as PACA-AU.
+
+This release includes:
+
+- APGI-AU: 84 previously released donors now have new clinical data
```
Comment (mcourtot) [inline docs/release-notes/data-releases.md:19]:
```suggestion
- APGI-AU: New clinical data has been added for 84 donors
```

File: docs/release-notes/data-releases.md [hunk 5]
```diff
@@ -6,13 +6,27 @@ platform_key: DOCS_DATA_RELEASES
...
+- PACA-CA: 38 previously released donors now have new clinical data
```
Comment (mcourtot) [inline docs/release-notes/data-releases.md:20]:
```suggestion
- PACA-CA: New clinical data has been added for 38 donors
```

File: docs/release-notes/data-releases.md [hunk 6]
```diff
@@ -6,13 +6,27 @@ platform_key: DOCS_DATA_RELEASES
...
+- PTC-SA: 239 previously released donors now have new clinical data
```
Comment (mcourtot) [inline docs/release-notes/data-releases.md:21]:
```suggestion
- PTC-SA: New clinical data has been added for 239 donors
```

**Pull request merged**